db/migrate/20260427130661_add_return_url_to_spree_adyen_payment_sessions.spree_adyen.rb

```
@@ -0,0 +1,16 @@
+# This migration comes from spree_adyen (originally 20250813152608)
+class AddReturnUrlToSpreeAdyenPaymentSessions < ActiveRecord::Migration[7.2]
+  def change
+    add_column :spree_adyen_payment_sessions, :return_url, :string
+
+    SpreeAdyen::PaymentSession.reset_column_information
+    Spree::Store.find_each do |store|
+      redirect_to = Spree::Core::Engine.routes.url_helpers.redirect_adyen_payment_session_url(host: store.url_or_custom_domain)
+      store.payment_methods.adyen.find_each do |gateway|
```

RuboCop (1.86.1)

[warning] Unused block argument - `gateway`. You can omit the argument if you don't care about it.

(Lint/UnusedBlockArgument)